swift/ArrowFlight/Sources/ArrowFlight/FlightClient.swift

```diff
@@ -28,6 +28,40 @@ public class FlightClient {
         client = Arrow_Flight_Protocol_FlightServiceAsyncClient(channel: channel)
     }
     
+    private func readMessages(_ responseStream: GRPCAsyncResponseStream<Arrow_Flight_Protocol_FlightData>) async throws -> ArrowReader.ArrowReaderResult {
+        let reader = ArrowReader()
+        let arrowResult = ArrowReader.MakeArrowReaderResult();
+        for try await data in responseStream {
+            switch reader.fromMessage(data.dataHeader, dataBody: data.dataBody, result: arrowResult) {
+            case .success(_):
+                continue;
+            case .failure(let error):
+                throw error
+            }
+        }
+        
+        return arrowResult;
+    }
+    
+    private func writeBatches(_ requestStream: GRPCAsyncRequestStreamWriter<Arrow_Flight_Protocol_FlightData>, descriptor: FlightDescriptor, recordBatchs: [RecordBatch]) async throws {
+        let writer = ArrowWriter()
+        switch writer.toMessage(recordBatchs[0].schema) {
+        case .success(let schemaData):
+            try await requestStream.send(FlightData(schemaData, dataBody: Data(), flightDescriptor: descriptor).toProtocol())
+            for rc in recordBatchs {
+                switch writer.toMessage(rc) {
```

Comment:
`rc` -> `rb` ("R"ecord "B"atch) or `rc` -> `recordBatch` may be better.

Reply:
Will do.

Reply:
@abandy Could you check this?

Reply:
Good catch, will update, thanks!